README.md

```diff
@@ -19,26 +20,129 @@ Most useful scripts for development:
 * `npm test` - fires up Jest test runner
 
 Other scripts which could be helpful:
+
 * `e2e:dev` - runs all e2e tests locally
 * `npm run start:build` -  starts a production version of app locally
 * `npm run test:update` - updates all tests' snapshots from scratch
 * `npm run format` - formats whole codebase with prettier
 
-## Conventions
+## Contributing
+
+We welcome all forms of contribution! Bugs reports :bug:, feedback and feature requests :pray: can be just as valuable as code contributions!
+
+Please direct all bug reports :bug:, feature requests :pray: or other feedback to project **Github Issues**. This allows the Daily Routine maintainers to effectively review, triage and prioritise any incoming work.
+
+If you want to make code contributions, please follow the code contribution process :computer:. Whether you have a specific fix :wrench: or feature in mind already, or if you have spare time and just want to help out - we'd love to hear from you! :muscle:
+
+If you want to make quotes contributions, please follow the quotes contribution process. Daily routine is a collection of absurd and funny, but still realistic quotes. The bigger amount of quotes, the better experience will be – let your imagination go wild! :dizzy:
+
+
+### :loudspeaker: Quotes contribution
```

Comment:
The description is detailed, but I worry that less technical (or even less JS-oriented) persons e.g: Project Managers gonna have a problem with contributing the way it's described 🤔.   It's a lot easier to edit `.json` file with plain text quotes rather than `.js` file in several places. Without IDE it's easy to break sth when attempting to add _a simple quote_.  Maybe we can add step-by-step example? If we make it easier for a random contributor we may increase overall contribute stats - just a thought. 🙂😎

Reply:
I think its fair remark.
I dont personally care too much about the container for data, it can be json, further it can be split to seperate files(but maybe not - usually I dislike editing several files, if they do not exceed 100 lines).

As for lack of IDE, I practically never use IDE and its very difficult to break and pass tests.
But the json suggestion I can look into, tomorrow.

Reply:
I would personally wait and see if we will have any contributors, and then adjust process in case of unclear instructions.